packages/file-search/src/browser/quick-file-open.ts

```diff
@@ -168,7 +168,10 @@ export class QuickFileOpenService implements QuickAccessProvider {
 
             if (location.uri.scheme === 'file' && !alreadyCollected.has(uriString) && fuzzy.test(fileFilter, uriString)) {
                 if (recentlyUsedItems.length === 0) {
-                    recentlyUsedItems.push({ type: 'separator', label: 'recently opened' });
+                    recentlyUsedItems.push({
+                        type: 'separator',
+                        label: nls.localizeByDefault('recently opened')
+                    });
```

Comment:
minor: splitting the lines are unnecessary
```suggestion
                    recentlyUsedItems.push({ type: 'separator', label: nls.localizeByDefault('recently opened') });
```

The same is true below.